src/providers/aptos.ts

```diff
@@ -40,6 +40,11 @@ export default class AptosProvider extends BloctoProvider implements AptosProvid
   constructor({ chainId, server, appId }: AptosProviderConfig) {
     super();
     invariant(chainId, "'chainId' is required");
+
+    if (!appId) {
+      console.error('[Warning] It will be necessary to interact with Blocto wallet via your app id in the near future.');
+    }
```

Comment:
I think we can use `invariant` directly

Reply:
69d138f6a96792807c18bf02397fa63c4ae31ddb updated